translations/translations.json

```
@@ -293,6 +293,17 @@
       "loading": "加载中。。。"
     }
   },
+  "cityNotCooperating": {
+    "de": {
+      "callToAction": "Schlag Integreat Deiner\nStadt vor!",
+      "explanation": "Das gemeinnützige Konzept der Integreat-App hat Dich überzeugt und Du glaubst, auch Deine Stadt, Dein Landkreis oder Deine Gemeinde könnte von der digitalen Lösung profitieren? Dann kannst Du hiermit ganz einfach aktiv werden.",
+      "whatToDo": "Das kannst Du tun:",
+      "findOutMail": "E-Mail Adresse Deiner Stadt herausfinden",
```

I think `Region` is the naming we use here.
```suggestion
      "findOutMail": "E-Mail Adresse Deiner Region herausfinden",
```